.github/workflows/build.yml

```
@@ -40,6 +40,17 @@ jobs:
       with:
         python-version: ${{ env.PYTHON_VERSION }}
 
+    - name: Manual Installation Steps (macOS)
+      if: ${{ runner.os == 'macOS' }}
+      run: |
+        #sudo rm -rf /Libary/Developer/CommandLineTools
+        #echo A | softwareupdate --install-rosetta
+        #arch -x86_64 xcode-select --install
+        arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+        export PATH="/usr/local/opt/node@16/bin:/usr/local/bin:$PATH"
+        arch -x86_64 brew install node@16 yarn git python@$PYTHON_VERSION
+        ln -s /usr/local/bin/python$PYTHON_VERSION /usr/local/bin/python
```

CI says

> `ln: /usr/local/bin/python: File exists`

So, we can add the `-f` flag that should force symlinking anyway, even if something exists at the link destination already?

```suggestion
        ln -sf /usr/local/bin/python$PYTHON_VERSION /usr/local/bin/python
```

Also: If this doesn't work, due to file permissions in `/usr/local/bin` dir, then maybe we would need to do it with `sudo`:

```suggestion
        sudo ln -sf /usr/local/bin/python$PYTHON_VERSION /usr/local/bin/python
```

---

Thanks a ton for this! Let me try what you suggested!